examples/web/nextjs-chat/src/app/api/momento/token/config.ts

```diff
@@ -23,11 +23,11 @@ import {
  *    export const tokenPermissions: TokenScope =  {
  *      permissions: [
  *        {
- *          role: CacheRole.ReadWrite | CacheRole.ReadOnly,
+ *          role: CacheRole.ReadWrite | CacheRole.ReadOnly | CacheRole.WriteOnly,
  *          cache: AllCaches | "your-cache-name"
  *        },
  *        {
- *          role: TopicRole.PublishSubscribe | TopicRole.SubscribeOnly,
+ *          role: TopicRole.PublishSubscribe | TopicRole.SubscribeOnly | TopicRole.PublishOnly,
```

Comment:
super nitpick but just FYI:

This wouldn't compile with the current examples, because they have a dependency on a released version of the SDK. Since they are just comments it's okay, but you will want to make sure that we update the examples deps to the new release of the JS SDK as soon as your changes are merged and we do a release.